readme.md

```diff
@@ -8,101 +8,145 @@
 [![Backers][backers-badge]][collective]
 [![Chat][chat-badge]][chat]
 
-[**rehype**][rehype] plugin to add metadata (Open Graph, Twitter) to your head.
+**[rehype][]** plugin to add metadata to the `<head>`.
 
 ## Contents
 
+*   [What is this?](#what-is-this)
+*   [When should I use this?](#when-should-i-use-this)
 *   [Install](#install)
 *   [Use](#use)
 *   [API](#api)
     *   [`unified().use(rehypeMeta[, options])`](#unifieduserehypemeta-options)
     *   [`Config`](#config)
 *   [Metadata](#metadata)
+*   [Examples](#examples)
+    *   [Example: frontmatter in markdown](#example-frontmatter-in-markdown)
+    *   [Example: inferring metadata](#example-inferring-metadata)
+*   [Types](#types)
+*   [Compatibility](#compatibility)
 *   [Security](#security)
 *   [Related](#related)
 *   [Contribute](#contribute)
 *   [License](#license)
 
-## Install
+## What is this?
+
+This package is a [unified][] ([rehype][]) plugin to manage the metadata (Open
+Graph, Twitter Cards, other SEO things) that can be found in `<head>`.
```

Comment:
This feels a bit off, meta is more than SEO.
Maybe reframing this as "this plugin adds SEO metadata, using the meta tag in the head of the document"?

Reply:
it does do more than just SEO. But it doesn’t do all metadata.

What if the word `other` is removed?

Reply:
Maybe, or adding an item to the list which is not SEO related